WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteFragment.kt

```diff
@@ -517,13 +518,21 @@ class MySiteFragment : Fragment(R.layout.my_site_fragment),
     }
 
     private fun MySiteFragmentBinding.loadData(cardAndItems: List<MySiteCardAndItem>) {
+        if(actionableEmptyView.isVisible) {
```

Comment:
Minor (🔍): The `if` needs a space before opening the parenthesis, that's why this PR fails its static analysis checks.

PS: Same applies for the below `loadEmptyView(...)` function.